src/views/List.jsx

```diff
@@ -40,34 +40,37 @@ export function List({ data, listPath }) {
 			) : null}
 			{listPath && data.length === 0 ? (
 				<>
-					<h2>
-						This list is currently empty. Click below to add your first item.
-					</h2>
-					<button onClick={() => handleClick('/manage-list')}>
-						Add first item
-					</button>
+					<AddItem data={data} listPath={listPath} />
+					<h2>This list is currently empty!</h2>
```

Comment:
Praise: I noticed that an empty list had this message earlier today, and I'm glad that you caught this!